metaflow/metaflow_config.py

```diff
@@ -254,5 +262,6 @@ def get_authenticated_boto3_client(module, params={}):
 
         session = boto3.Session(botocore_session=botocore_session, region_name=METAFLOW_AWS_S3_REGION)
         return session.client("s3")
+    """
```

Comment:
Please do not comment out code, either delete the functionality or handle it with conditionals.
Now that METAFLOW_AWS_ARN is no longer needed (deleting the functionality), please also remove all references to it in the code base

Reply:
Ok sounds good @talebzeghmi , this code is non-functional now (causes an error), so I'll remove it.

Reply:
Resolved.